apps/admin-x-activitypub/src/views/Profile/components/ProfilePage.tsx

```
@@ -0,0 +1,181 @@
+                                    <Icon className='opacity-0 transition-opacity group-hover/handle:opacity-100' name='arrow-top-right' size='xs'/>
+                                </a>
+                                {(account.bio || customFields.length > 0) && (<div ref={contentRef} className={`ap-profile-content transition-max-height relative text-[1.5rem] duration-300 ease-in-out [&>p]:mb-3 ${isExpanded ? 'max-h-none pb-7' : 'max-h-[160px] overflow-hidden'} relative`}>
+                                    {!isLoadingAccount ?
+                                        <div dangerouslySetInnerHTML={{__html: account?.bio ?? ''}} /> :
+                                        <>
+                                            <Skeleton />
+                                            <Skeleton className='w-full max-w-48' />
+                                        </>
+                                    }
+                                    {customFields.map((attachment: {name: string, value: string}) => (
+                                        <span className='mt-3 line-clamp-1 flex flex-col text-[1.5rem]'>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)